ExpressWash/ExpressWash/Car+CoreDataProperties.swift

```diff
@@ -18,6 +18,7 @@ extension Car {
 
     @NSManaged public var color: String
     @NSManaged public var carId: Int32
+    @NSManaged public var clientId: Int16
```

Comment:
`userId`s are Int32s. It would make more sense to keep them the same here so they don't have to be cast every time.

Reply:
Also it's a bad idea to use 16-bit integers for IDs in general because you max out at 65,000ish and for a nation-wide app I think you could reasonably hope to get more users than that.